cub/cub/util_device.cuh

```diff
@@ -70,7 +70,7 @@ CUB_NAMESPACE_BEGIN
  * \brief Empty kernel for querying PTX manifest metadata (e.g., version) for the current device
  */
 template <typename T>
-__global__ void EmptyKernel(void) { }
+CUB_DETAIL_KERNEL_ATTRIBUTES void EmptyKernel(void) { }
```

Comment:
Question: do we still need the empty kernel to query for the version of the current device? If so, why? And if not: would be great if it could be removed.

Reply:
I don't think we still need empty kernel. Changing the underlying mechanism is going to be a part of the following issue: https://github.com/NVIDIA/cub/issues/556.